src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -0,0 +1,190 @@
+                            'ezimage',
+                            'ezembed',
+                            'eztable',
+                            ...this.customTags,
+                        ],
+                        tabIndex: 2,
+                    },
+                    styles: {
+                        selections: [
+                            ...this.customTagsToolbars,
+                            new window.eZ.ezAlloyEditor.ezLinkConfig(),
+                            new window.eZ.ezAlloyEditor.ezTextConfig({ customStyles: this.customStylesConfigurations }),
+                            new window.eZ.ezAlloyEditor.ezParagraphConfig({ customStyles: this.customStylesConfigurations }),
+                            new window.eZ.ezAlloyEditor.ezFormattedConfig({ customStyles: this.customStylesConfigurations }),
+                            new window.eZ.ezAlloyEditor.ezCustomStyleConfig({ customStyles: this.customStylesConfigurations }),
+                            new window.eZ.ezAlloyEditor.ezHeadingConfig({ customStyles: this.customStylesConfigurations }),
+                            new window.eZ.ezAlloyEditor.ezTableConfig(),
+                            new window.eZ.ezAlloyEditor.ezEmbedImageConfig(),
+                            new window.eZ.ezAlloyEditor.ezEmbedConfig(),
+                        ],
+                        tabIndex: 1,
+                    },
+                },
+                extraPlugins:
+                    AlloyEditor.Core.ATTRS.extraPlugins.value +
+                    ',' +
+                    ['ezaddcontent', 'ezmoveelement', 'ezremoveblock', 'ezembed', 'ezfocusblock', 'ezcustomtag'].join(','),
+            });
+
+            const wrapper = this.getHTMLDocumentFragment(container.closest('.ez-data-source').querySelector('textarea').value);
+            const section = wrapper.childNodes[0];
+
+            if (!section.hasChildNodes()) {
+                section.appendChild(document.createElement('p'));
+            }
+
+            alloyEditor.get('nativeEditor').setData(section.innerHTML);
+
+            container.addEventListener('blur', (event) => {
+                const data = alloyEditor.get('nativeEditor').getData();
+                const doc = document.createDocumentFragment();
+                const root = document.createElement('div');
+
+                root.innerHTML = data;
+                doc.appendChild(root);
+
+                [...doc.querySelectorAll('[data-ezelement="ezembed"]')].forEach(this.emptyEmbed);
+                [...doc.querySelectorAll('[data-ezelement="ezcustomtag"]')].forEach(this.emptyEmbed);
+
+                event.target.closest('.ez-data-source').querySelector('textarea').value = this.xhtmlify(root.innerHTML).replace(
+                    this.xhtmlNamespace,
+                    this.ezNamespace
+                );
+            });
+
+            return alloyEditor;
+        }
+    };
+
+    eZ.BaseRichtText = BaseRichtText;
```

Comment:
Maybe we should use `eZ.addConfig` here?

Reply:
I want to be in line with the rest of the AE files.